deployer/src/deployer/upload.py

```diff
@@ -603,6 +626,42 @@ def on_task_complete(task):
                 if delta.days < 30:
                     continue
 
+            # Remember, if `key` is from a "index.html" file it will be represented
+            # something like this: `main/en-us/docs/web/api/documentorshadowroot`
+            # with the prefix and the `/index.html` portion removed.
```

Comment:
I think the `with the prefix` phrase should be removed right? So this should be:
```suggestion
            # with the `/index.html` portion removed.
```